### ISSUE TITLE
Password Reset flow via vector.im has a broken link to a Riot logo

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-web#9073]
![image](https://user-images.githubusercontent.com/1922197/53951141-b1f6fe80-40c5-11e9-8820-8b6ff0326e80.png)

We moved https://riot.im/app/themes/riot/img/logos/riot-logo.svg

**Comment by jryans** [comment]
Duplicate of #9006. Waiting on updated Sydent to be deployed.
